config.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import "github.com/googlecloudplatform/gcsfuse/internal/config"
+
+func overrideWithLoggingFlags(mountConfig *config.MountConfig, flags *flagStorage) {
```

Comment:
can we move everything into internal/config package and also merge config.go into util.go file.